src/device/wg_controller.rs

```diff
@@ -1,7 +1,6 @@
 use super::{Entities, RequestedState, Result};
 use ipnetwork::IpNetwork;
-use std::collections::HashMap;
-use std::collections::{BTreeMap, HashSet};
+use std::collections::{BTreeMap, HashMap, HashSet};
```

Comment:
HashMap is added here, but not used in wg_controller. Or am I missing something here ?

Reply:
It is used, I just moved it so there's one less use statement